### PR TITLE
GP2-1398_comtrade_data_copy

### PR DESCRIPTION
**Pull request by bobmeredith**
Moves a copy of filtered comtrade data into the directory-api database.
The new request to get comtrade data supports multiple countries, single commodity code and does no processing, so all the data are returned for the FE to compute changes and trends.
The country list is always a list of iso2 codes.
There is also a new request for country data that allows multiple country's data to be requested in one go, again using iso3. The model names required can be passed in so this can become a generic request to get any data for a country.

A compacted comtrade data extract is in S3.  run `make manage import_comtrade_data` to load it.
This process can take an hour or more I'm afraid.

The extract is generated as follows:
Comtrade data are downloaded as vast csv files (about 6GB), one for each year. There is a management command 
`make manage import_comtrade_data -- --raw  <filename>`
That reads through such files and loads your db with only the data that are needed.
Then, a sql dump of the table to generate the single data file that gets uploaded to S3.

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +23,7 @@
 - GP2-1382 - getting paid structure
 - GP2-1180 - travel bus BE
 - NOTICKET - Exportplan make dict default for JSON fields.
-- GP2-1181 - business risk
+- GP2-1181 - 1398business risk
```
Comment (depsiatwal) [inline CHANGELOG.md:26]: look like a typo here

**Review by depsiatwal — APPROVED** [review]
Small comment just on CL
nicely done :-)

File: dataservices/tests/conftest.py [hunk 2]
```diff
@@ -16,3 +16,53 @@ def internet_usage_data():
...
+    )
+    models.ComtradeReport.objects.create(
+        country=countries['NL'],
+        country_iso3='NLD',
+        year=2020,
+        commodity_code='123456',
+        trade_value=91.97,
+    )
+    yield
+    models.InternetUsage.objects.all().delete()
```
Comment (webbyfox) [inline dataservices/tests/conftest.py:46]: I guess you need to delete all records from `ComtradeReport` here

File: dataservices/tests/conftest.py [hunk 3]
```diff
@@ -16,3 +16,53 @@ def internet_usage_data():
...
+        year_2019=12,
+    )
+    models.EaseOfDoingBusiness.objects.create(
+        country=countries['NL'],
+        country_code='NL',
+        country_name='Netherlands',
+        year_2019=13,
+    )
+    yield
+    models.InternetUsage.objects.all().delete()
```
Comment (webbyfox) [inline dataservices/tests/conftest.py:67]: similarly here `EaseOfDoingBusiness`

**Review by webbyfox — CHANGES_REQUESTED** [review]
minor comments

File: dataservices/tests/conftest.py (hunk 2 above)
Comment (bobmeredith) [reply dataservices/tests/conftest.py:46]: Hehe - well spotted - thanks

File: CHANGELOG.md (hunk 1 above)
Comment (bobmeredith) [reply CHANGELOG.md:26]: Thanks for the spot.

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**